specification/azurelargeinstance/resource-manager/Microsoft.AzureLargeInstance/preview/2023-07-20-preview/azurelargeinstance.json

```diff
@@ -0,0 +1,1312 @@
+        "tags": [
+          "Azure Large Storage Instances"
+        ],
+        "operationId": "AzureLargeStorageInstance_Get",
+        "summary": "Gets an Azure Large Storage Instance.",
+        "description": "Gets an Azure LargeStorage Storage instance for the specified subscription, resource group, and instance name.",
+        "x-ms-examples": {
+          "Get an AzureLargeStorageStorage instance": {
+            "$ref": "./examples/AzureLargeStorageInstance_Get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AzureLargeStorageInstanceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AzureLargeStorageInstance"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Azure Large Storage Instances"
+        ],
+        "summary": "Creates an Azure Large Storage Instance.",
+        "operationId": "AzureLargeStorageInstance_Create",
+        "x-ms-examples": {
+          "Put a new AzureLargeStorageInstance": {
+            "$ref": "./examples/AzureLargeStorageInstance_Create.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/AzureLargeStorageInstanceNameParameter"
+          },
+          {
+            "in": "body",
+            "name": "requestBodyParameters",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AzureLargeStorageInstance"
+            },
+            "description": "request body for put call"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Update ok. Successfully created an Azure LargeStorageStorage resource.",
```

Comment:
200 OK is for updated, not created, but you use the word created here, as if 200 would be the final response for a Create operation. That should not be the case!